### ISSUE TITLE
[Feature Request] Lutris launcher

### ISSUE THREAD

**Issue by rodneyrod** [CodingKoopa/lucas-simpsons-hit-and-run-mod-launcher-linux-launcher#4]
Lutris is a game launcher for Linux that's sort of like PlayOnLinux but also for emulators and native games. In theory the games are portable across linux distros as a result.
They currently have a launcher for Simpsons Hit and Run but getting the Lucas Launcher working in that environment has proved difficult for me.

Due to the limitations of the original game and the improvements the Lucas Launcher brings, I'd like to request that a launcher for Lutris be made to bring those new features over to Linux in more ways.

**Comment by CodingKoopa** [comment]
> Lutris is a game launcher for Linux that's sort of like PlayOnLinux but also for emulators and native games. In theory the games are portable across linux distros as a result.

I've heard good things about Lutris. At one point, [I pursued making an AppImage](https://github.com/CodingKoopa/lucas-simpsons-hit-and-run-mod-launcher-linux-launcher/issues/3), but, given the [current state](https://github.com/AppImage/AppImageKit/wiki/Bundling-Windows-applications#ongoing-research) of Wine support, I agree that it might be a better idea to make a Winetricks verb and/or Lutris launcher.

> They currently have a launcher for Simpsons Hit and Run but getting the Lucas Launcher working in that environment has proved difficult for me.

Oh, I wasn't aware of this, I'll check it out and possible merge that script with this repo.

**Comment by rodneyrod** [comment]
Oh, right, yes the launchers at the moment aren't for the PC version, but I manually created one for the PC version in Lutris.
I think the Lucas launcher is falling over on a missing dependency but I can't figure out what.

**Comment by CodingKoopa** [comment]
Yep. Wine's built in Mono implementation is not sufficient to run the mod launcher, so you need to install the native .NET 3.5 SP1 runtime with `winetricks`.

**Comment by rodneyrod** [comment]
I ended up trying your method recently but now it crashes with the following error:L

https://imgur.com/a/wUMEZRf

**Comment by rodneyrod** [comment]
@CodingKoopa Did I grab all the required libraries or do I need more than .net 3.5 sp1?

**Comment by CodingKoopa** [comment]
That's quite a strange error, I've never seen that before. .NET 3.5 SP1 definitely should be sufficient. Out of curiosity, does installing [Wine Mono](https://wiki.winehq.org/Mono) change anything?
